update-keycloak-user-group/src/main/resources/application.conf

```diff
@@ -0,0 +1,9 @@
+keycloak {
+  url = "https://auth.tdr-"${?ENVIRONMENT}".nationalarchives.gov.uk"
```

Comment:
Will that work for prod? The url is: `auth.tdr.national.gov.uk for prod`.

Reply:
No, it won't, missed that. I've changed the environment variable to be SUB_DOMAIN so you can set it as tdr-integration or tdr.